cogs/image_prompt_optimizer.py

```diff
@@ -34,11 +34,12 @@ def __init__(
         self.deletion_queue = deletion_queue
 
         try:
+            image_pretext_path = self.converser_cog.data_path / "image_optimizer_pretext.txt"

```

Comment:
Neat, didn’t know you could do this

Reply:
I love pathlib. Remove os.path from your life today!

Reply:
Consider it done! :)